scripts/windows/build/Publish-Branch.ps1

```diff
@@ -0,0 +1,276 @@
+<#
```

Comment:
Remove. Windows containers aren't supported in 1.2.

Reply:
Removed.